### PR TITLE
add stateful and pagination wrapper controller templates

### PR DESCRIPTION
**Pull request by petropavel13**

### CODE REVIEW

File: touchin-pagination/Code/View/view_controller.swift.liquid [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import LeadKit
+import TableKit
+
+final class {{ module_info.name }}ViewController: UIViewController {
+
+    var viewModel: {{ module_info.name }}ViewModel!
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    private lazy var tableDirector = {
```
Comment (igorkislyuk) [inline touchin-pagination/Code/View/view_controller.swift.liquid]: Could be shorter.

**Review by igorkislyuk — CHANGES_REQUESTED** [review]

File: touchin-pagination/Code/View/view_controller.swift.liquid [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+final class {{ module_info.name }}ViewController: UIViewController {
+
+    var viewModel: {{ module_info.name }}ViewModel!
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    private lazy var tableDirector = TableDirector(tableView: tableView)
+
+    private lazy var paginationWrapper = {
```
Comment (scoreyou) [inline touchin-pagination/Code/View/view_controller.swift.liquid:13]: same about shortening

Comment (petropavel13) [reply touchin-pagination/Code/View/view_controller.swift.liquid:13]: Too long
screen shot 2018-03-27 at 17 52 09
or ugly
screen shot 2018-03-27 at 17 52 31

what did you prefer? :)

Comment (scoreyou) [reply touchin-pagination/Code/View/view_controller.swift.liquid:13]: :)

**Review by scoreyou — APPROVED** [review]

**Review by igorkislyuk — APPROVED** [review]

**Pull request merged**